app/src/hooks/disconnectedWeb3.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { ethers } from 'ethers'
+import { INFURA_PROJECT_ID } from '../common/constants'
+
+export interface DisconnectedWeb3Context {
+  account: string
```

Comment:
If `account` is going to be the empty string, I would just remove it. It also makes more sense if you are connecting via Infura.

Reply:
Done, I remove it